tests/setup_windows_spatialite.py

```diff
@@ -0,0 +1,15 @@
+import urllib.request
+import zipfile
+from os.path import join, dirname
+import os
+
+pth = 'https://github.com/AequilibraE/aequilibrae/releases/download/V0.6.0.post1/mod_spatialite-NG-win-amd64.zip'
```

Comment:
where did you get this zip from?
I had real problems with the version i found - I had to replace the c++ std runtime dll files with up to date versions :/
https://github.com/sqlitebrowser/sqlitebrowser/wiki/SpatiaLite-on-Windows 
Did you have any such problems?

Reply:
I got it directly from here.  http://www.gaia-gis.it/gaia-sins/windows-bin-NEXTGEN-amd64/

Just uploaded to the repository as a way to make sure all tests work in case the original url fails (It is linked in the documentation)